lambda_backend/predict_microservice/base_inference.py

```diff
@@ -0,0 +1,144 @@
+import cv2
+import numpy as np
+from configparser import RawConfigParser
+
+DEF_SCORE = 0.3
+DEF_NMS = 0.4
```

Comment:
Hmm, maybe we should use more descriptive names?

Reply:
To add on, it would be better to use DEFAULT instead of DEF. When I first read this I thought DEF was DEFINE